pkgs/applications/audio/termusic/default.nix

```diff
@@ -1,31 +1,38 @@
 { lib
 , stdenv
 , rustPlatform
-, fetchCrate
-, fetchpatch
+, fetchFromGitHub
+, protobuf
+, dbus
+, openssl
 , pkg-config
 , alsa-lib
 , darwin
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "termusic";
-  version = "0.7.10";
+  version = "v0.7.11";
 
-  src = fetchCrate {
-    inherit pname version;
-    hash = "sha256-m0hi5u4BcRcEDEpg1BoWXc25dfhD6+OJtqSZfSdV0HM=";
+  src = fetchFromGitHub {
+    owner = "tramhao";
+    repo = pname;
```

Comment:
`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
DRY = Don't Repeat Yourself